addons/sale/controllers/portal.py

```diff
@@ -140,20 +140,25 @@ def portal_order_page(
             if session_obj_date != today:
                 # store the date as a string in the session to allow serialization
                 request.session['view_quote_%s' % order_sudo.id] = today
+                subtype_id = request.env.ref('sale.mt_order_viewed')
+                follower_salesman = order_sudo.message_follower_ids.filtered(
+                    lambda follower: follower.partner_id.ids == order_sudo.user_id.partner_id.ids)
+
+                if subtype_id.id in follower_salesman.subtype_ids.ids:
                 # The "Quotation viewed by customer" log note is an information
                 # dedicated to the salesman and shouldn't be translated in the customer/website lgg
```

Comment:
```suggestion
                    # The "Quotation viewed by customer" log note is an information
                    # dedicated to the salesman and shouldn't be translated in the customer/website lgg
```
Move comments inside as well